docker-compose.yml

```
@@ -22,14 +22,31 @@ services:
     user: $USER
     command: scripts/build.sh
 
+  build-x86_64-unknown-linux-gnu:
+    build:
+      context: .
+      dockerfile: scripts/ci-docker-images/builder-x86_64-unknown-linux-gnu/Dockerfile
+    environment:
+      NATIVE_BUILD: "false"
+      TARGET: x86_64-unknown-linux-gnu
+      FEATURES: default
+      CARGO_TERM_COLOR: always
+    volumes:
+      - $PWD:$PWD
+      - ./target/x86_64-unknown-linux-gnu/cargo/registry:/usr/local/cargo/registry
+      - ./target/x86_64-unknown-linux-gnu/cargo/git:/usr/local/cargo/git
+      - ./target/x86_64-unknown-linux-gnu/rustup/tmp:/usr/local/rustup/tmp
+    working_dir: $PWD
```

```suggestion
    working_dir: $PWD
    user: $USER
```

Our builds can run as non-root now